src/main/java/com/taxi/model/Car.java

```diff
@@ -0,0 +1,72 @@
+package com.taxi.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private HashSet<Driver> driverSet;
```

Comment:
Всегда используем интерфейсы
```suggestion
    private Set<Driver> drivers;
```